lib/screens/navigate/dropdown_inventory.dart

```diff
@@ -8,10 +8,11 @@ class DropDown extends StatefulWidget {
 
 class _DropDownState extends State<DropDown> {
   List<ListItem> _dropdownItems = [
-    ListItem(1, "Select Your Audit Report "),
-    ListItem(2, "Second Item"),
-    ListItem(3, "Third Item"),
-    ListItem(4, "Fourth Item")
+    ListItem(1, "Single "),
+    ListItem(2, "Married"),
```

Comment:
all hard coded should be moved a single configuration file. it should not be in code at all

Reply:
I designed static Dropdown , @deepaketech make me understand How to do.???